modules/content/widgets/zig-zag.php

```diff
@@ -269,16 +269,16 @@ private function add_style_zigzags_section() {
 				'label' => esc_html__( 'Align First Graphic', 'hello-plus' ),
 				'type' => Controls_Manager::CHOOSE,
 				'options' => [
-					'left' => [
-						'title' => esc_html__( 'Left', 'hello-plus' ),
+					'start' => [
+						'title' => esc_html__( 'Start', 'hello-plus' ),
 						'icon' => 'eicon-order-start',
```

Comment:
Maybe something like this:

```suggestion
						'icon' => 'eicon-order-' . ( is_rtl() ? 'end' : 'start' ),
```

For better RTL support.